src/Runner.Worker/JobExtension.cs

```diff
@@ -325,6 +328,11 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    if (jobContext.Global.Variables.GetBoolean("__DISK.CHECK") ?? true)
```

Comment:
Is this convention? Do we have other variables like this? 

I think we should save this off as a constant somewhere and explain where it comes from in the code.